imap_processing/ialirt/l0/parse_mag.py

```diff
@@ -338,6 +341,53 @@ def calibrate_and_offset_vectors(
     return calibrated[:, :3]
 
 
+def transform_to_inertial(
+    sc_spin_phase_rad: np.ndarray,
+    sc_inertial_right: np.ndarray,
+    sc_inertial_decline: np.ndarray,
+    mag_vector: np.ndarray,
+) -> np.ndarray:
+    """
+    Transform vector to ECLIPJ2000.
+
+    Parameters
+    ----------
+    sc_spin_phase_rad : numpy.ndarray
+        Spin phase for 4 packets 0 to 2π radians, shape (4).
+    sc_inertial_right : numpy.ndarray
+        Inertial right ascension for 4 packets 0 to 2π radians, shape (4).
+    sc_inertial_decline : numpy.ndarray
+        Inertial right ascension for 4 packets -π/2 to π/2 radians, shape (4).
```

Comment:
Is this comment correct? 
```suggestion
        Inertial decline for 4 packets -π/2 to π/2 radians, shape (4).
```